sample/www/jxcore/test/thaliscenarios.js

```diff
@@ -0,0 +1,79 @@
+var test = require('tape');
+var ThaliEmitter = require('../thali/thaliemitter');
+
+test('ThaliEmitter can call startBroadcasting and endBroadcasting without error', function (t) {
+  var e = new ThaliEmitter();
+
+  console.log('about to call startBroadcasting');
+  e.startBroadcasting((+ new Date()).toString(), 5000, function (err1) {
```

Comment:
For hygiene sake please define the port using a variable. Preferably at a global level in another file so that all the tests that need a local PouchDB instance can share the same one. Prevents silly errors.